### DISCUSSION TITLE
Results for 14 pretrained models coming soon

### DISCUSSION THREAD

**Discussion by alasdairforsythe** [alasdairforsythe/tokenmonster#16] [Announcements]
I've been pretraining from scratch NanoGPT Small and Medium sized models with different TokenMonster vocabularies, and comparing against GPT-2 Tokenizer and TikToken p50k_base.

Results will be up in 1-2 weeks.

**Comment by SSamDav** [comment]
Any news about this experiment? 👀

**Reply by abb128** [reply]
https://github.com/alasdairforsythe/tokenmonster/blob/main/benchmark/pretrain.md

**Comment by Sovenok-Hacker** [comment]
> I've been pretraining from scratch NanoGPT Small and Medium sized models with different TokenMonster vocabularies, and comparing against GPT-2 Tokenizer and TikToken p50k_base.
> 
> Results will be up in 1-2 weeks.

Hooray!!!

**Comment by adi-kmt** [comment]
Any follow ups on [this](https://github.com/alasdairforsythe/tokenmonster/blob/main/benchmark/pretrain.md) @alasdairforsythe ? For 80% one-word tokens and 20% multi-word tokens?
